.storybook/config.js

```diff
@@ -51,6 +51,7 @@ configure(() => {
   require('./../packages/bpk-component-mobile-scroll-container/stories');
   require('./../packages/bpk-component-modal/stories');
   require('./../packages/bpk-component-nudger/stories');
+  require('./../packages/bpk-component-sliding-drawer/stories');
```

Comment:
Can we position this alphabetically please? (below `bpk-component-slider`)

Reply:
It was there from when it was called `overflow-menu`.

My IDE took care of the renaming so it was left where it was.

I'll move it when we settle on a name.